src/main/java/org/sonarlint/intellij/config/project/ProjectExclusionsPanel.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * SonarLint for IntelliJ IDEA
+ * Copyright (C) 2015 SonarSource
+ * sonarlint@sonarsource.com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonarlint.intellij.config.project;
+
+import com.intellij.openapi.project.Project;
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+import javax.swing.JComponent;
+import javax.swing.JPanel;
+import org.sonarlint.intellij.config.ConfigurationPanel;
+
+public class ProjectExclusionsPanel implements ConfigurationPanel<SonarLintProjectSettings> {
+  private static final String EMPTY_LABEL = "No exclusions configured";
+  private final Project project;
+
+  private ExclusionTable table;
+  private JPanel tablePanel;
+  private JPanel panel;
+
+  public ProjectExclusionsPanel(Project project) {
+
+    this.project = project;
+  }
+
+  private void createUIComponents() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "createUIComponents" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)